app/views/offers/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<h1><%= @offer.user.username %>'s Offer</h1>
+<em>Coins: <%= @offer.coins %></em>
+<% if @players.empty? %>
+  There are no players associated with this trade
+<% else %>
+  <h2>Players</h2>
+  <table>
```

Comment:
tables definitely get the job done, but I have found that using divs makes it a lot easier to tweak the formatting or completely redesign something in the future without having to throw everything away.

This also will allow you to write modular CSS  such that your classes help provide context to other people reading your code.  If you are interested, here is an introductory [article](http://inlehmansterms.net/2014/08/25/modular-css-with-suitcss/) on how we write modular Sass at CustomInk with SUIT CSS